packages/CollapsibleChecklists/src/components/Group/Group.styles.js

```diff
@@ -1,27 +1,34 @@
-import tokens from "@paprika/tokens";
 import { css } from "styled-components";
-import { disabledStyle } from "../helpers.styles";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+import { disabledStyles } from "../helpers.styles";
 
 const groupStyles = css`
   ${({ isDisabled }) => {
     return `
       [data-pka-anchor="collapsible.heading"] {
-        align-items: center;
+        align-items: flex-start;
         background-color: ${tokens.color.cremeLighten5};
         border-bottom: 1px solid ${tokens.color.blackLighten60};
         display: flex;
-        padding: ${tokens.spaceSm};
+        padding: ${tokens.spaceSm} ${tokens.spaceSm} ${tokens.spaceSm} ${tokens.space};
 
-        input[type="checkbox"] {
-          margin-right: ${tokens.space};
+        ${isDisabled ? disabledStyles : ""};
+
+        .collapsible__label {
+          flex-shrink: 0;
         }
 
         label {
-          align-items: center;
+          ${stylers.lineHeight(-1)}
+          align-items: flex-start;
           display: flex;
+          margin-top: 2px;
         }
 
-        ${isDisabled ? disabledStyle : ""};
+        input[type="checkbox"] {
+          margin: 0.4em ${tokens.space} 0 ${tokens.spaceSm}
```

Comment:
more `em`s

Reply:
I await your counter-counter-argument to my previous counter-argument 😄